__tests__/app.test.js

```diff
@@ -45,7 +45,50 @@ describe("/api", () => {
         expect(Object.keys(response.body.api)).toEqual([
           "GET /api",
           "GET /api/topics",
+          "GET /api/articles",
         ]);
       });
   });
 });
+
+describe("/api/articles/:articles_id", () => {
+  test("GET 200: receive a the article requested by their respective id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
```

Comment:
Well done, this is a really important thing to check and make sure the article id is what you want it to be!